From a1a10eab2b44dee2c10d0864961d9966733f1215 Mon Sep 17 00:00:00 2001
From: Shubh <shubhaggarwalva@gmail.com>
Date: Tue, 15 Sep 2026 12:59:46 +0530
Subject: [PATCH] JusBrowse: fix resource links after logo purge

- mipmap-anydpi/ic_launcher*.xml monochrome -> ic_launcher_foreground (deleted ic_launcher_monochrome)
- view_crash_reporter: fenix_error_1 -> ic_launcher_foreground
- search_widget_info: fenix_search_widget -> ic_launcher_foreground
---
 .../fenix/app/src/main/res/layout/view_crash_reporter.xml       | 2 +-
 .../fenix/app/src/main/res/mipmap-anydpi/ic_launcher.xml        | 2 +-
 .../fenix/app/src/main/res/mipmap-anydpi/ic_launcher_round.xml  | 2 +-
 .../android/fenix/app/src/main/res/xml/search_widget_info.xml   | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/mobile/android/fenix/app/src/main/res/layout/view_crash_reporter.xml b/mobile/android/fenix/app/src/main/res/layout/view_crash_reporter.xml
index fd94e64e4c49..2e1a8875909d 100644
--- a/mobile/android/fenix/app/src/main/res/layout/view_crash_reporter.xml
+++ b/mobile/android/fenix/app/src/main/res/layout/view_crash_reporter.xml
@@ -20,7 +20,7 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintWidth_percent="0.8"
-        app:srcCompat="@drawable/fenix_error_1" />
+        app:srcCompat="@drawable/ic_launcher_foreground" />
 
     <TextView
         android:id="@+id/title"
diff --git a/mobile/android/fenix/app/src/main/res/mipmap-anydpi/ic_launcher.xml b/mobile/android/fenix/app/src/main/res/mipmap-anydpi/ic_launcher.xml
index de34ac4858cb..feb8882d51ea 100644
--- a/mobile/android/fenix/app/src/main/res/mipmap-anydpi/ic_launcher.xml
+++ b/mobile/android/fenix/app/src/main/res/mipmap-anydpi/ic_launcher.xml
@@ -5,5 +5,5 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@color/ic_launcher_background"/>
     <foreground android:drawable="@drawable/ic_launcher_foreground"/>
-    <monochrome android:drawable="@drawable/ic_launcher_monochrome"/>
+    <monochrome android:drawable="@drawable/ic_launcher_foreground"/>
 </adaptive-icon>
diff --git a/mobile/android/fenix/app/src/main/res/mipmap-anydpi/ic_launcher_round.xml b/mobile/android/fenix/app/src/main/res/mipmap-anydpi/ic_launcher_round.xml
index de34ac4858cb..feb8882d51ea 100644
--- a/mobile/android/fenix/app/src/main/res/mipmap-anydpi/ic_launcher_round.xml
+++ b/mobile/android/fenix/app/src/main/res/mipmap-anydpi/ic_launcher_round.xml
@@ -5,5 +5,5 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@color/ic_launcher_background"/>
     <foreground android:drawable="@drawable/ic_launcher_foreground"/>
-    <monochrome android:drawable="@drawable/ic_launcher_monochrome"/>
+    <monochrome android:drawable="@drawable/ic_launcher_foreground"/>
 </adaptive-icon>
diff --git a/mobile/android/fenix/app/src/main/res/xml/search_widget_info.xml b/mobile/android/fenix/app/src/main/res/xml/search_widget_info.xml
index 7ea72ca6d754..dc36262d8d49 100644
--- a/mobile/android/fenix/app/src/main/res/xml/search_widget_info.xml
+++ b/mobile/android/fenix/app/src/main/res/xml/search_widget_info.xml
@@ -9,7 +9,7 @@
     android:minHeight="40dp"
     android:resizeMode="horizontal"
     android:minResizeWidth="30dp"
-    android:previewImage="@drawable/fenix_search_widget"
+    android:previewImage="@drawable/ic_launcher_foreground"
     android:updatePeriodMillis="3600000"
     android:initialLayout="@layout/search_widget_large"
     android:widgetCategory="home_screen">
-- 
2.55.0

